core/network/warp/protocol.hpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_NETWORK_WARP_PROTOCOL_HPP
+#define KAGOME_NETWORK_WARP_PROTOCOL_HPP
+
+#include "network/common.hpp"
+#include "network/helpers/scale_message_read_writer.hpp"
+#include "network/impl/protocols/request_response_protocol.hpp"
+#include "network/warp/cache.hpp"
+
+namespace kagome::network {
+  using _WarpProtocol = RequestResponseProtocol<primitives::BlockHash,
```

Comment:
```suggestion
  using WarpProtocolType = RequestResponseProtocol<primitives::BlockHash,
```
Not mandatory. It seems that we did not use the leading underscore for names before.